src/bp/sessions_bp.py

```diff
@@ -0,0 +1,75 @@
+import random
+from flask import Blueprint, request
+from sqlalchemy import func
+from .home_bp import Sessions, db
+from .err_bp import handle_bad_input_501, handle_server_500
+
+# validates that curr_session_id in db:
+def session_id_validation(curr_session_id):
+    return db.session.query(db.exists().where(Sessions.id == curr_session_id)).scalar()
+
+def get_new_session_id():
+    
+    # generate random int with exact 9 integers to be a session_id:
+    gen_rand_session = random.randrange(100000000, 999999999, 9)
+
+    # validate that gen_rand_session not exists in the db:
+    while db.session.query(db.exists().where(Sessions.id == gen_rand_session)).scalar():
+         gen_rand_session = random.randrange(100000, 999000, 6)
+
```

Comment:
You can leave just the while somehow (do while would be classic)
It's ok like that anyway

Reply:
👍

Reply:
Found a bug there (different constants used)